mailscanner/functions.php

```diff
@@ -4644,7 +4644,7 @@ function validateInput($input, $type)
                 return true;
             }
         case 'maxmind':
-            if (preg_match('/^[0-9A-Za-z]{16}$/', $input)) {
+            if (preg_match('/^[0-9A-Za-z]{12,16}$/', $input)) {
```

Comment:
This should be `^([0-9A-Za-z]{12}|[0-9A-Za-z]{16})$` to catch 12 and 16 only
https://regex101.com/r/v4qDCx/1/

Reply:
Done.